lib/util/TestArea.cpp

```diff
@@ -15,6 +15,9 @@
    See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
    for more details.
 */
+
+#ifndef _MSC_VER // Currently not in Visual Studio build
```

Comment:
What is the purpose of this? Does the `TestArea.cpp` not compile with MSVC?

Reply:
I got linker errors when building TestArea.cpp ( undefined symbol test_area_* ), but these went away when turning off ERT_BUILD_CXX, so I will revert these changes.

Reply:
Good!